build.gradle

```diff
@@ -303,8 +332,15 @@ dependencies {
 
   functionalTestImplementation sourceSets.main.runtimeClasspath
   functionalTestImplementation sourceSets.test.runtimeClasspath
+
+  ccdTestImplementation sourceSets.main.runtimeClasspath
+  ccdTestImplementation sourceSets.test.runtimeClasspath
+
   functionalTestImplementation group: 'uk.gov.hmcts.reform', name: 'document-management-client', version: '7.0.0'
   functionalTestImplementation group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: versions.s2sClient
+
+  ccdTestImplementation 'com.github.hmcts:rse-cft-lib:0.1.0'
```

Comment:
If this is on the classpath it boots up CCD & Access management using spring autoconfiguration